### DISCUSSION TITLE
Can I open the Output Window as a split rather than a floating window?

### DISCUSSION THREAD

**Discussion by armanschwarz** [nvim-neotest/neotest#56] [Q&A] [answered]
Sometimes it would be nice to see the test error output while browsing through the file. Is this possible? Currently, when I run a test, I just get the popup which disappears when I navigate away.

**Comment by rcarriga** [comment, marked as answer]
I've added a new option `open_win` to the `output.open` function so you can use it to customise how the window is opened
```lua
require("neotest").output.open({ open_win = function() vim.cmd("vsplit") end })
```

**Reply by petobens** [reply]
Hi @rcarriga, is it possible to add the same `open_win` for the `attach()` function? Thanks!

**Comment by armanschwarz** [comment]
Works great. Thanks

**Comment by mirror-health-labs** [comment]
Neovim allows you to navigate to a pane by number with, e.g. `3<C-w><C-w>` to navigate to the third pane. Pane number corresponds to the layout of the panes in the current window.

When a floating window pops up, it seems to always be assigned to be the last pane. If you enter a larger number than there are panes, Neovim will take you to the last pane. So `9<C-w><C-w>` will take you to the floating pane as long as you had fewer than 9 panes open before the floating pane popped up. 

To make things a bit easier, I've added the following mapping to my `init.lua` to jump to the floating pane:

```lua
vim.api.nvim_set_keymap('n', '<C-w><C-f>', '9<C-w><C-w>', { noremap = true, silent = true })
```

This lets you navigate around the error message, but still closes the pane when you navigate away, which I find convenient.
